content/contribute/contributing-and-managing-patterns.adoc

```diff
@@ -0,0 +1,145 @@
+---
+menu: contribute
+title: Contributing and managing patterns
+weight: 50
+aliases: /contributing-and-managing-patterns/
+---
+
+:toc:
+:_content-type: ASSEMBLY
+include::modules/comm-attributes.adoc[]
+
+[id="contributing to patterns"]
+= Contributing to patterns
+
+To contribute, add the upstream repository as an additional remote and work on a branch other than main. This approach allows you to merge changes into the main branch (reflected in GitOps workflows) and makes upstream contributions easier. Contributions from your forked main branch typically include:
+
+. Customizations to `values-global.yaml` and other installation-specific files.
+. Commits made by Tekton and other automated processes unique to your setup.
+
+To isolate changes for upstream contributions (`hcp` is "Validated Patterns"), you can use a separate remote and branch name:
```

Comment:
Can we separate these commands into separate commands with lead in instruction: Run the following command to ...  easier to copy into command line plus there is some context on what you are doing